plugins/woocommerce-admin/client/marketplace/components/tabs/tabs.tsx

```diff
@@ -115,6 +121,11 @@ const renderTabs = (
 					key={ tabKey }
 				>
 					{ tabs[ tabKey ]?.title }
+					{ tabs[ tabKey ]?.show_badge && (
+						<span className="awaiting-mod">
+							{ tabs[ tabKey ]?.badge_text_to_render }
+						</span>
```

Comment:
Any reason not to use `<Badge />` component from `@woocommerce/components` ?

```
{ tabs[ tabKey ]?.badge && (
	<Badge
		key={ `${ tabKey }-badge` }
		count={ badge }
	/>
) }
```

Using this component makes it looks consistent with other elements in WooCommerce though the default styling is gray, we can override it by passing `className`.

Reply:
Thanks @gedex for this suggestion 
addressed [f9cad5d](https://github.com/woocommerce/woocommerce/pull/46034/commits/f9cad5df3065aa49a529bf1f554b9ad16101383b)